Linkbrary/javascript/register.js

```diff
@@ -0,0 +1,16 @@
+// 회원가입 시도
+function RegisterTry(event) {
+  event.preventDefault();
+  if (idError===false && pwError===false && pwcheckError===false) { //성공
```

Comment:
주석달아주신 부분을 실제 코드에 반영하면 더 가독성이 좋아질 거 같아요!
```suggestion
  const isSuccess = !idError&& !pwError && !pwcheckError
  if (isSuccess) {
```